src/seraphis_main/txtype_squashed_v1.cpp

```diff
@@ -635,12 +640,29 @@ bool validate_txs_batchable<SpTxSquashedV1>(const std::vector<const SpTxSquashed
         if (!tx)
             return false;
 
+        // used to gather all new tx commitments (sp enote image masked, then enote outputs)
+        rct::keyV &curr_tx_commitments = tools::add_element(tx_commitments);
+        curr_tx_commitments.reserve(tx->sp_input_images.size() + tx->outputs.size());
+
         // gather membership proof pieces
         for (const SpMembershipProofV1 &sp_membership_proof : tx->sp_membership_proofs)
             sp_membership_proof_ptrs.push_back(&sp_membership_proof);
 
+        // gather enote images and also their masked commitments (mult 1/8)
         for (const SpEnoteImageV1 &sp_input_image : tx->sp_input_images)
+        {
             sp_input_image_ptrs.push_back(&(sp_input_image.core));
+            curr_tx_commitments.push_back(rct::scalarmultKey(masked_commitment_ref(sp_input_image),
+                rct::INV_EIGHT));
+        }
+
+        // make keyV for all output commitments (mult 1/8)
+        for (const SpEnoteV1 &sp_output_enote : tx->outputs)
+            curr_tx_commitments.push_back(rct::scalarmultKey(sp_output_enote.core.amount_commitment,
+                rct::INV_EIGHT));
```

Comment:
```suggestion
        // make keyV for all output commitments (mul 1/8)
        for (const SpEnoteV1 &sp_output_enote : tx->outputs)
        {
            curr_tx_commitments.push_back(rct::scalarmultKey(sp_output_enote.core.amount_commitment,
                rct::INV_EIGHT));
        }
```
Braces if your block-scope is multiple lines.